cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -348,7 +348,23 @@ if(PARQUET_BUILD_BENCHMARKS)
 endif()
 
 ## Apache Arrow
-find_package(Arrow)
+pkg_check_modules(ARROW arrow)
+pkg_check_modules(ARROW_IO arrow-io)
+if (ARROW_FOUND AND ARROW_IO_FOUND)
+  set(ARROW_INCLUDE_DIR ${ARROW_INCLUDE_DIRS})
+  set(ARROW_LIBS ${ARROW_LIBRARY_DIRS})
+
+  set(ARROW_LIB_NAME libarrow)
+  set(ARROW_IO_LIB_NAME libarrow_io)
```

Comment:
OK. I changed to use `CMAKE_SHARED_LIBRARY_PREFIX`.
I borrowed the code from `FindArrow.cmake`. We may need to do the same change in `FindArrow.cmake` later.